src/objects/celery.py

```diff
@@ -82,6 +82,21 @@ def receiver_setup_logging(
                     "handlers": ["console"],
                     "level": "INFO",
                 },
+                "celery": {
```

Comment:
Hmm this does make the configuration diverge quite a bit from Open Forms, which should work the same and doesn't add these extra loggers (https://github.com/open-formulieren/open-forms/blob/master/src/openforms/celery.py#L76).

Could you check if Open Forms also does not output duplicate logs and see what the configuration difference is that causes this issue in Objects?